code/obj/machinery/law_racks.dm

```diff
@@ -317,6 +317,13 @@
 					M.moustache_mode = 1
 					user.visible_message("<span class='alert'><b>[user.name]</b> uploads a moustache to [M.name]!</span>")
 					M.update_appearance()
+		else if (istype(I, /obj/item/peripheral/videocard))
+			var/obj/item/peripheral/videocard/V = I
+			if (GET_COOLDOWN(src, "mine_cooldown") == 0)
```

Comment:
this syntax makes me inherently annoyed but it makes sense here so. FINE

Reply:
is the better way to do this with `!GET_COOLDOWN(...)`?

Reply:
no, imo this is clearer and I think it's the right choice.
just seeing `if (x == 0)` is hair-raising.